packages/protocol/scripts/bash/release-on-devchain.sh

```diff
@@ -47,11 +47,12 @@ yarn run truffle exec ./scripts/truffle/verify-bytecode.js --network development
 
 echo "- Check versions of current branch"
 # From check-versions.sh
+
 echo " - Checkout migrationsConfig.js at $BRANCH"
 git checkout $BRANCH -- migrationsConfig.js
 
-CONTRACT_EXCLUSION_REGEX=".*Test|Mock.*|I[A-Z].*|.*Proxy|MultiSig.*|ReleaseGold|SlasherUtil|UsingPrecompiles"
-yarn ts-node scripts/check-backward.ts sem_check --old_contracts $BUILD_DIR/contracts --new_contracts build/contracts --exclude $CONTRACT_EXCLUSION_REGEX --output_file report.json
```

Comment:
is it intentional that this check-backward.ts check was also removed here?

Reply:
It wasn't removed, it was moved to bash/contract-exclusion-regex.sh to avoid duplicated code

Reply:
Maybe I'm missing something, but I don't see the second line
```
yarn ts-node scripts/check-backward.ts sem_check --old_contracts $BUILD_DIR/contracts --new_contracts build/contracts --exclude $CONTRACT_EXCLUSION_REGEX --output_file report.json
```
in the `contract-exclusion-regex.sh` file?

Reply:
oh you're right, deleted one more line than I should have, nice catch!